assets/sass/components/_pagination.scss

```diff
@@ -0,0 +1,22 @@
+nav.row.siblings {
+    display: flex;
+    flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	margin: 0 5px;
+
+	a.nav-flex-left{
+		justify-content: flex-start;
```

Comment:
Firefox shows me a warning that `justify-content` has no effect since the element is not a flex container. Is it better to just remove that rule since everything works fine without it? Firefox suggests adding `display: flex` or `display: grid`, but it doesn't look like we would need that.

Reply:
Yes, it is true. The `justify-content` property is not necessary.